lib/ts/types.ts

```diff
@@ -261,5 +261,6 @@ export type PreAPIHookFunction = (context: {
 export type PostAPIHookFunction = (context: {
     requestInit: RequestInit;
     url: string;
-    response: Response;
+    fetchResponse: Response;
```

Comment:
Post API hook should either:
- Clone the response and give it to the post API hook (so that the user can consume the response as well as the auth react code can consume it)
- Or, return the {jsonBody: .., fetchResposne:... } object
- You should also add a test related to post API hook

Reply:
The post api hook in web-js (since that is where it will be triggered from) clones the response object and passes it